README.md

```diff
@@ -308,7 +308,7 @@ You still can use `wrapper_class` option to set only a css class. This is just a
 Our select helper accepts the same arguments as the [default Rails helper](http://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-select). Here's an example of how you pass both options and html_options hashes:
 
 ```erb
-<%= f.select :product, [["Apple", 1], ["Grape", 2]], { label: "Choose your favorite fruit:" }, { class: "selectpicker",  wrapper: { class: 'has-warning', data: { foo: 'bar' } } } %>
+<%= f.select :product, [["Apple", 1], ["Grape", 2]], { label: "Choose your favorite fruit:", wrapper: { class: 'has-warning', data: { foo: 'bar' } } }, { class: "selectpicker" } %>
```

Comment:
Should `data: { foo: 'bar' }` also be part of the HTML options? E.g. `{ label: "Choose your favorite fruit:", wrapper: { class: 'has-warning' } }, { class: "selectpicker", data: { foo: 'bar' } }`.

Reply:
Ignore my previous comment. I misunderstood your intent for the `data` option.